harness/src/network.rs

```diff
@@ -150,13 +150,21 @@ impl Network {
     /// Read out all messages generated by peers in the `Network`.
     ///
     /// Note: messages are not filtered by any configured filters.
+    #[inline]
     pub fn read_messages(&mut self) -> Vec<Message> {
         self.peers
             .iter_mut()
             .flat_map(|(_peer, progress)| progress.read_messages())
             .collect()
     }
 
+    #[inline]
```

Comment:
@nrc advises us in the PingCAP style guide that this is generally unnecessary https://pingcap.github.io/style-guide/rust/functions.html#functions

Reply:
Good advice! I'll remove the `#[inline]` involved in this pr. And maybe later I can post a PR to remove them all.

Reply:
@Hoverbear I heard about that removing `#[inline]` might hurt the performance in practice 😂.